DiceKeys/Views/Screens/SecretsView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  SecretsView.swift
+//  DiceKeys
+//
+//  Created by Angelos Veglektsis on 7/6/22.
+//
+
+import SwiftUI
+
+struct SecretsView: View {
+    @EnvironmentObject var derivationRecipeStore: DerivationRecipeStore
+    
+    let navigateTo: (DiceKeyPresentPageContent) -> Void
+    
+    var body: some View {
+        
+        List{
+            if(derivationRecipeStore.savedDerivationRecipes.count > 0){
+                Section(header: Text("Saved Recipes")) {
+                    ForEach(derivationRecipeStore.savedDerivationRecipes) { recipe in
+                        HStack{
+                            Text(recipe.name)
+                            Spacer()
+                        }
+                        .contentShape(Rectangle())
+                        .onTapGesture(perform: {
+                            navigateTo(.Derive(DerivationRecipeBuilderType.recipe(recipe)))
+                        })
+                    }
+                }
+            }
+
+            Section(header: Text("Build-in Recipes")) {
```

Comment:
Should be "Built-in Recipes" (note the T at the end of built)

Reply:
Fixed